### PR TITLE
Nits from incredible squaring rs 

### PR DESCRIPTION
**Pull request by supernovahs**
This PR contains latest code that is being pulled by incredible squaring rs repo . 

This includes 
- Small nits
- Bug fixes that we come across  while building IS
- other features

goal is to get it reviewed continuously till  IS is complete . Then merge this as well

### CODE REVIEW

File: crates/chainio/clients/elcontracts/src/reader.rs [hunk 1]
```diff
@@ -303,7 +302,7 @@ mod tests {
         let service_manager_address = anvil_constants::get_service_manager_address().await;
         let service_manager_contract = mockAvsServiceManager::new(
             service_manager_address,
-            anvil_constants::ANVIL_RPC_URL.clone(),
+            get_provider(&"http://localhost:8545"),
```
Comment (dralves) [inline crates/chainio/clients/elcontracts/src/reader.rs:305]: why not use a constant

File: crates/chainio/clients/elcontracts/src/writer.rs [hunk 2]
```diff
@@ -150,6 +150,7 @@ impl ELChainWriter {
             .el_chain_reader
             .get_strategy_and_underlying_erc20_token(strategy_addr)
             .await;
+        println!("tokens result {:?}",tokens_result);
```
Comment (dralves) [inline crates/chainio/clients/elcontracts/src/writer.rs:153]: hum, println in the middle of prod code?

File: crates/services/avsregistry/src/chaincaller.rs [hunk 3]
```diff
@@ -53,10 +47,13 @@ impl AvsRegistryService for AvsRegistryServiceChainCaller {
         let mut operators_avs_state: HashMap<FixedBytes<32>, OperatorAvsState> = HashMap::new();
 
         let operators_stakes_in_quorums = self
-            .avs_registry
+            .operators_info_service
+            .avs_registry_reader
             .get_operators_stake_in_quorums_at_block(block_num, Bytes::from(Vec::from(quorum_nums)))
             .await?;
 
+        println!("operator stakes in quorums {:?}",operators_stakes_in_quorums);
```
Comment (dralves) [inline crates/services/avsregistry/src/chaincaller.rs:55]: should this be a debug statement

File: crates/services/avsregistry/src/chaincaller.rs [hunk 4]
```diff
@@ -101,19 +98,20 @@ impl AvsRegistryService for AvsRegistryServiceChainCaller {
...
             .get_operators_avs_state_at_block(block_num, quorum_nums)
             .await?;
-
         Ok(quorum_nums
             .iter()
             .map(|quorum_num| {
                 let mut pub_key_g1 = G1Projective::from(PublicKey::identity());
                 let mut total_stake: U256 = U256::from(0);
                 for operator in operators_avs_state.values() {
+                    println!("quorum_num {:?}",quorum_num);
```
Comment (dralves) [inline crates/services/avsregistry/src/chaincaller.rs:107]: same, here and elsewhere

File: examples/info-operator-service/src/main.rs [hunk 5]
```diff
@@ -1,60 +1,62 @@
...
-use eigen_services_operatorsinfo::operatorsinfo_inmemory::OperatorInfoServiceInMemory;
-use std::sync::Arc;
-use tokio::sync::watch;
-use tokio::sync::Mutex;
-use tokio::task;
-use tokio::time::{self, Duration};
-const HOLESKY_PROVIDER: &str = "https://ethereum-holesky.blockpi.network/v1/rpc/public";
-const WS_HOLESKY_PROIVIDER: &str = "wss://holesky.drpc.org";
-use eigen_logging::get_test_logger;
+// use alloy_primitives::address;
```
Comment (dralves) [inline examples/info-operator-service/src/main.rs:1]: this seems messy

**Review by dralves — CHANGES_REQUESTED** [review]

File: crates/services/avsregistry/src/chaincaller.rs (hunk 3 above)
Comment (supernovahs) [reply crates/services/avsregistry/src/chaincaller.rs:55]: my bad , forgot to remove these prints.

File: examples/info-operator-service/src/main.rs (hunk 5 above)
Comment (supernovahs) [reply examples/info-operator-service/src/main.rs]: done

**Comment by supernovahs** [comment]
pending merge from main

**Review by dralves — APPROVED** [review]

**Pull request merged**